src/legacy/core_plugins/kibana/public/context/index.js

```diff
@@ -27,6 +27,7 @@ import './app';
 import contextAppRouteTemplate from './index.html';
 import { getRootBreadcrumbs } from '../discover/breadcrumbs';
 import { npStart } from 'ui/new_platform';
+import { subscribeWithScope } from 'ui/utils/subscribe_with_scope';
```

Comment:
can you pls explain what this is for ?

Reply:
Oh, this is a dark place.
This is the way to subscribe to observables from Angular scope.
Courtesy of @spalger